frontend/src/views/Register.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <v-container fluid fill-height>
+    <v-layout align-center justify-center>
+      <v-flex xs12 sm8 md4>
+        <v-card>
+          <v-card-title>
+            <v-toolbar-title>Create account</v-toolbar-title>
+          </v-card-title>
+          <v-card-text>
+              <v-form @submit.prevent="register()" v-model="valid">
+
+                  <v-text-field
+                    v-model="email"
+                    :rules="emailRules"
+                    label="E-mail"
+                    required
+                  ></v-text-field>
+
+                  <v-text-field
+                    v-model="password"
+                    label="Password"
+                    type="password"
+                    required
+                  ></v-text-field>
+
+                  <v-text-field
+                    v-model="cpassword"
+                    label="Confirm Password"
+                    type="password"
+                    required
```

Comment:
Since you added the `passwordShow` variable, look at `src/views/Login.vue` for an example on how to use it to show the password conditionally.